src/main/config/index.ts

```diff
@@ -20,3 +23,23 @@ export const LIMITED_NAME_REGISTERED_WITH_AML = `${BASE_LIMITED_TRADER_URL}/name
 export const LIMITED_BUSINESS_MUSTBE_AML_REGISTERED = `${BASE_LIMITED_TRADER_URL}/business-mustbe-aml-registered/business-mustbe-aml-registered`;
 export const UNINCORPORATED_WHAT_IS_THE_BUSINESS_NAME = `${BASE_UNINCORPORATED_URL}/what-is-the-business-name/what-is-the-business-name`;
 export const LIMITED_COMPANY_INACTIVE = `${BASE_LIMITED_TRADER_URL}/company-inactive/company-inactive`;
+export const ACSP_SERVICE_HOST = "localhost";
```

Comment:
All environment related config(API_URL, CHS_API_KEY etc.) can be moved to utils/properties.ts